pkg/kubelet/checkpoint/checkpoint_test.go

```diff
@@ -103,22 +103,22 @@ func TestWriteLoadDeletePods(t *testing.T) {
 					t.Errorf("expected %#v, \ngot %#v", p.pod, lpod)
 				}
 			} else {
-				t.Errorf("Got unexpected result for %v, should have been loaded", pname)
+				t.Errorf("Got unexpected result for %s, should have been loaded", pname)
 			}
 		} else if lpod != nil {
-			t.Errorf("Got unexpected result for %v, should not have been loaded", pname)
+			t.Errorf("Got unexpected result for %s, should not have been loaded", pname)
```

Comment:
`t.Errorf("Got unexpected result for pod %q, should not have been loaded", pname)`